swervelib/Mk4iSwerveModuleHelper.java

```diff
@@ -358,16 +537,49 @@ public static SwerveModule createFalcon500Neo(
         return new SwerveModuleFactory<>(
                 gearRatio.getConfiguration(),
                 getFalcon500DriveFactory(configuration),
-                getNeoCanCoderSteerFactory(configuration)// ADD HELIUM LOGIC HERE
+                getNeoHeliumSteerFactory(configuration)
         ).create(
                 driveMotorPort,
                 new SteerConfiguration<>(
                         steerMotorPort,
-                        new CanCoderAbsoluteConfiguration(steerEncoderPort, steerOffset)
+                        new CanandCoderAbsoluteConfiguration(steerEncoderPort)
+                )
+        );
+    }
+
+    /**
+     * Creates a Mk4i swerve module that uses a Kraken X60 for driving and a NEO for steering.
+     *
+     * @param configuration    Module configuration parameters to use.
+     * @param gearRatio        The gearing configuration the module is in.
+     * @param driveMotorPort   The CAN ID of the drive Kraken X60.
+     * @param steerMotorPort   The CAN ID of the steer NEO.
+     * @param steerEncoderPort The CAN ID of the steer CANCoder.
+     * @param steerOffset      The offset of the CANCoder in radians.
+     * @return The configured swerve module.
+     */
+    public static SwerveModule createKrakenX60Neo(
+            Mk4ModuleConfiguration configuration,
+            GearRatio gearRatio,
+            int driveMotorPort,
+            int steerMotorPort,
+            int steerEncoderPort,
+            double steerOffset
+    ) {
+        return new SwerveModuleFactory<>(
+                gearRatio.getConfiguration(),
+                getKrakenX60DriveFactory(configuration),
+                getNeoHeliumSteerFactory(configuration)
+        ).create(
+                driveMotorPort,
+                new SteerConfiguration<>(
+                        steerMotorPort,
+                        new CanandCoderAbsoluteConfiguration(steerEncoderPort)
```

Comment:
This method and others with a neo steer motor needs to check if the neo uses a helium encoder or a cancoder